sotodlib/preprocess/processes.py

```diff
@@ -647,11 +651,13 @@ def check_frequency_cutoff(fmin, fmax):
             wn_f_low = self.calc_cfgs.get("low_f", 5)
             wn_f_high = self.calc_cfgs.get("high_f", 10)
             wn_f_high = check_frequency_cutoff(wn_f_low, wn_f_high)
+            std_units = self.calc_cfgs.get("std_units", False)
             wn = tod_ops.fft_ops.calc_wn(aman, pxx=pxx,
                                          freqs=psd.freqs,
                                          nseg=psd.get('nseg'),
                                          low_f=wn_f_low,
-                                         high_f=wn_f_high)
+                                         high_f=wn_f_high,
+                                         std_units=std_units)
```

Comment:
Why change fft_ops.calc_wn? Why not just change this calc step in preprocess and wrap into calc_aman both wn and std?